flybywiresim/a32nx
# feat: elec system sounds and other huge changes

## Summary of Changes


I have changed many many sounds in this PR, I will list them here:

- I have changed the avionics fan sound to be more realistic using higher quality recordings from komp
- Changed the engine start sounds to have a more realistic sounding fuel introduction for a cold start (when the crank phase is skipped, it introduces fuel at a lower n2 percentage which i simulated)
- Elongated the engine shutdown sound
- Added the power transfer sounds when switching from buses
- When turning on batteries, there is a clunk sound for one battery and a quieter clunk sound for both batteries when they are on, which is then followed by the light transformer whines and the CIDS chimes - this means that all of the sounds now follow the correct electric bus
- The whines that the elec system makes are all improved too
- The cabin ambience is improved also where the cabin fans now come from around the leading edge of the wing as that is where they are located. Now the light transformers are also heard inside the cabin and there is a small whine that comes from the avionics fan that is heard if you pay attention in the front to mid cabin
- There are now pack and brake fan spool down sounds
- Made the exterior idle be heard from farther away

## Screenshots (if necessary)



## References





## Additional context


Known issues:
-When the power up sequence is interrupted, like the avionics ventilation test, the sound will keep playing even when cold and dark, this is due to a wwise limitation of the sim now allowing wwise stop events to play and I can not do anything about this currently

-When the left engine is shut down the aircraft loses and gains power making many sounds at once play - this is not an issue with sounds and with the elec system - this will be fixed by @davidwalschots soon in a future PR


Discord username (if different from GitHub):

## Testing instructions


- Bat 1 and 2 to on and listen to the new clunks and whines
- Start APU and turn off ext pwr to listen to power transfer
- Listen to the new avionics fan sound
- Turn on APU bleed and turn it off - then go to exterior view to listen to the shutdown
- Turn brake fan on and go to exterior view to turn it off to listen to the shutdown 
- Go to front/mid cabin (showcase view 13) and listen to the improved ambience
- Start engine and listen to new fuel intro - listen to exterior for the first eng start and interior for the second eng start
- Check that exterior idle is heard from farther away (1000m instead of 500m)
- Shut down engines and listen from interior


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
